apache/mod_perl2.sls

```diff
@@ -17,9 +17,9 @@ mod-perl2:
       - service: apache
 
 {% if grains['os_family']=="Debian" %}
-a2enmod perl2:
+a2enmod perl:
```

Comment:
Let's not change the state identifier unnecessarily:

```suggestion
a2enmod perl2
```

Reply:
@myii If I reset the identifier, the state will fail again. So we need the `name` argument like that:

```
a2enmod perl2:
  cmd.run:
    - name: a2enmod perl
    - unless: ls /etc/apache2/mods-enabled/perl.load
    - order: 225
    - require:
      - pkg: mod-perl2
    - watch_in:
      - module: apache-restart
    - require_in:
      - module: apache-restart
      - module: apache-reload
      - service: apache
```
Am I right?

Reply:
@SynPrime You're absolutely right, when I ran the tests, I didn't change the state identifier.  In this case, it's being used implicitly by `cmd.run`, so it _is_ required to be changed in this case!  Let's consider this resolved and merge this PR.  Thanks again.